x-pack/plugins/security_solution/public/cases/components/create/index.test.tsx

```diff
@@ -161,381 +77,89 @@ const fillForm = async (wrapper: ReactWrapper) => {
-            <Create />
-          </Router>
-        </TestProviders>
-      );
+    await waitFor(() => expect(mockHistory.push).toHaveBeenNthCalledWith(1, '/basic-case-id'));
+  });
 
-      await fillForm(wrapper);
-      await act(async () => {
-        await wrapper.find(`[data-test-subj="create-case-submit"]`).first().simulate('click');
-        wrapper.update();
-        expect(
-          wrapper.find(`[data-test-subj="create-case-loading-spinner"]`).exists()
-        ).toBeTruthy();
-      });
+  it('it should insert a timeline', async () => {
+    let attachTimeline = noop;
+    useInsertTimelineMock.mockImplementation((value, onTimelineAttached) => {
+      attachTimeline = onTimelineAttached;
     });
 
-    it('it should select the default connector set in the configuration', async () => {
-      useCaseConfigureMock.mockImplementation(() => ({
-        ...useCaseConfigureResponse,
-        connector: {
-          id: 'servicenow-1',
-          name: 'SN',
-          type: ConnectorTypes.servicenow,
-          fields: null,
-        },
-        persistLoading: false,
-      }));
-
-      useConnectorsMock.mockReturnValue({
-        ...sampleConnectorData,
-        connectors: connectorsMock,
-      });
+    const wrapper = mount(
+      <TestProviders>
+        <Router history={mockHistory}>
+          <Create />
+        </Router>
+      </TestProviders>
+    );
 
-      const wrapper = mount(
-        <TestProviders>
-          <Router history={mockHistory}>
-            <Create />
-          </Router>
-        </TestProviders>
-      );
-
-      await fillForm(wrapper);
+    act(() => {
       wrapper.update();
-
-      await act(async () => {
-        wrapper.find(`[data-test-subj="create-case-submit"]`).first().simulate('click');
-      });
-
-      await waitFor(() =>
-        expect(postCase).toBeCalledWith({
-          ...sampleData,
-          connector: {
-            fields: {
-              impact: null,
-              severity: null,
-              urgency: null,
-            },
-            id: 'servicenow-1',
-            name: 'My Connector',
-            type: '.servicenow',
-          },
-        })
-      );
+      attachTimeline('[title](url)');
     });
 
-    it('it should default to none if the default connector does not exist in connectors', async () => {
-      useCaseConfigureMock.mockImplementation(() => ({
-        ...useCaseConfigureResponse,
-        connector: {
-          id: 'not-exist',
-          name: 'SN',
-          type: ConnectorTypes.servicenow,
-          fields: null,
-        },
-        persistLoading: false,
-      }));
-
-      useConnectorsMock.mockReturnValue({
-        ...sampleConnectorData,
-        connectors: connectorsMock,
-      });
-
-      const wrapper = mount(
-        <TestProviders>
-          <Router history={mockHistory}>
-            <Create />
-          </Router>
-        </TestProviders>
-      );
-
-      await fillForm(wrapper);
+    await waitFor(() => {
```

Comment:
we do not need the waitFor or updates. Hard to do suggestion over these lines but...
```
    act(() => {
      attachTimeline('[title](url)');
    });
    expect(wrapper.find(`[data-test-subj="caseDescription"] textarea`).text()).toBe('[title](url)');
```